README.md

```diff
@@ -162,7 +162,7 @@ To extend the instrumentation with the OpenTelemetry Instrumentation for Java,
 you have to use a compatible API version.
 
 <!-- IMPORTANT: do not change comments or break those lines below -->
-The Splunk Distribution of OpenTelemetry Java version <!--SPLUNK_VERSION-->1.14.0<!--SPLUNK_VERSION--> is compatible
+The Splunk Distribution of OpenTelemetry Java version <!--SPLUNK_VERSION-->1.14.2<!--SPLUNK_VERSION--> is compatible
 with:
 
 * OpenTelemetry API version <!--OTEL_VERSION-->1.15.0<!--OTEL_VERSION-->
```

Comment:
shouldn't this and the otel instrumentation version be 1.16.0?

Reply:
Yeah, everything was a version behind...guessing it got overlooked/skipped last release. Will update.